tools/create-yorkie-app/frameworks.ts

```diff
@@ -1,12 +1,17 @@
 import { cyan, lightGreen, reset, yellow } from 'kolorist';
 
-type ColorFunc = (str: string | number) => string;
 export type Framework = {
   name: string;
   display: string;
   color: ColorFunc;
   variants: Array<FrameworkVariant>;
 };
+
+/**
+ * @see https://github.com/marvinhagemeister/kolorist#readme
+ */
+type ColorFunc = (str: string | number) => string;
+
```

Comment:
I would suggest moving the `ColorFunc` type declaration above the `Framework` type declaration.

Why: When reading the `Framework` type declaration, there is no information about the `ColorFunc` type.